### DISCUSSION TITLE
Generate routes definition for easier redirect and goto

### DISCUSSION THREAD

**Discussion by chientrm** [sveltejs/kit#6500] [Ideas]
Generate `.svelte-kit/routes.ts` from the dir tree `src/routes` so that it's easier to `redirect` at server-side and `goto` at client-side and less likely to get to the wrong route.

[Example routes.ts](https://github.com/chientrm/podie/blob/main/src/lib/constants/routes.ts)

**Comment by smblee** [comment]
I use [ts-routes](https://github.com/leancodepl/ts-routes) for this. Would definitely be nice for autogeneration, but it also hasn't been too much hassle to manage on my own with this lib.
